### PR TITLE
fix: always show date select

### PR DESCRIPTION
**Pull request by kelatev**
image

### CODE REVIEW

**Comment by gihan9a** [comment]
Please merge this ASAP. We are seeing this unexpected selected date on UI

**Comment by martijnrusschen** [comment]
Can you provide more information on the change?

**Comment by kelatev** [comment]
1. I added new prop "showDateSelect". When this prop is append (for example: <DatePicker showDateSelect selected={utils.newDate()} />), block ".react-datepicker__aria-live" is showing "Selected date"
2. I fixed block ".react-datepicker__aria-live" when they empty they not showing full

File: src/index.jsx [hunk 1]
```diff
@@ -1076,7 +1080,7 @@ export default class DatePicker extends React.Component {
             locale,
           }
         )}`;
-      } else {
+      } /*if (this.props.showDateSelect)*/ else {
```
[the remark this answers is not in the document]
Comment (kelatev) [reply src/index.jsx:1083]: This hint may be helping to understand when all "else"

**Comment by literalpie** [comment]
Do I understand right that this change removed aria announcements unless you opt in with the new input? Was this change made based on a misunderstanding of [this issue](https://github.com/Hacker0x01/react-datepicker/issues/3924)? (the correct solution is to add the CSS styles mentioned in that conversation)

If the CSS class is causing too much difficulty, maybe inline styles for the aria live announcement would be a better approach? (I can make a PR for that if people think it makes sense)

**Pull request merged**